content/docs/command-reference/version.md

```diff
@@ -12,20 +12,18 @@ usage: dvc version [-h] [-q | -v]
-| `Python version`                            | Version of Python used in the environment where DVC is initialized                                                                                                    |
-| `Platform`                                  | Information about the operating system of the machine                                                                                                                 |
-| [`Binary`](#what-we-mean-by-binary)         | Shows whether DVC was installed from a package or from a binary release                                                                                               |
-| `Package`                                   | Name of the package manager used to install DVC if any (`pip`, `conda`, etc)                                                                                          |
-| `Supported remotes`                         | Remote types that have all needed dependencies installed for                                                                                                          |
-| `Cache`                                     | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported between the <abbr>workspace</abbr> and the <abbr>cache</abbr> |
-| `Filesystem type`                           | Shows the filesystem type (eg. ext4, FAT, etc.) and mount point of the cache and <abbr>workspace</abbr> directories                                                   |
-| `Repo`                                      | Shows whether we are in a DVC repo and/or Git repo                                                                                                                    |
-
-> If `dvc version` is executed outside a DVC project, no `Cache` info is
-> printed, and the `Filesystem type` output is of the current working directory.
+| Line                                        | Detail                                                                                                                                                               |
+| ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`DVC version`](#components-of-dvc-version) | Version of DVC, and name of the binary or package manager (`pip`, `conda`, etc.) used to install DVC (along with a Git commit hash in case of a development version) |
+| `Platform`                                  | Version of Python in the environment where DVC is initialized, and information about the operating system                                                            |
+| `Supports`                                  | Type of [remote storages](/doc/command-reference/remote/add#supported-storage-types) that are supported (their required dependencies are installed)                  |
```

Comment:
- Type of remote storages -> Types of remote storage
- that are supported -> "supported by this instance of DVC"? Something like that